lncfg/db.go

```diff
@@ -211,6 +216,80 @@ func (db *DB) GetBackends(ctx context.Context, chanDBPath,
 			),
 			Replicated: true,
 		}, nil
+
+	case PostgresBackend:
+		postgresBackend, err := kvdb.Open(
+			kvdb.PostgresBackendName, ctx,
+			db.Postgres, "channeldb",
```

Comment:
Yeah, I guess we need constants for those names. Going to add them in my PR.

Reply:
Used constant in this PR